tests/actions.go

```diff
@@ -3601,6 +3628,22 @@ func (oa *OperatorActions) WaitForDmClusterReady(dc *v1alpha1.DMCluster, timeout
 	})
 }
 
+func (oa *OperatorActions) WaitForDmClusterDeleted(ns, dcName string, timeout, pollInterval time.Duration) error {
```

Comment:
do not log, but return the reason about why check failed when timeout?

Reply:
:+1: updated in https://github.com/pingcap/tidb-operator/pull/3982/commits/3c184d2f443b957cd5a873ef74416730c1928fac.